packages/mobile/src/screens/chat-screen/ChatMessageListItem.tsx

```diff
@@ -311,7 +311,9 @@ export const ChatMessageListItem = memo(function ChatMessageListItem(
                     >
                       <UserGeneratedText
                         variant='body'
+                        lineHeight='multi'
                         color={isAuthor ? 'staticWhite' : 'default'}
+                        textAlign='left'
```

Comment:
why is `textAlign` required now?

Reply:
oh i removed the text-align left css module i believe

Reply:
should we make left the default so we don't have to specify every time?

Reply:
yeah fwiw i think it is the default unless flex specifies center, so i think we are generally good to remove this